CHANGES.md

```diff
@@ -4,7 +4,16 @@ Libplanet changelog
 Version 0.25.5
 --------------
 
-To be released.
+Released on February 18, 2022.
+
+ -  Fixed `HashDigest<T>.DeriveFrom(ReadOnlySpan<byte>)` method's bug where
+    it had thrown `IndexOutOfRangeException` for the input longer or shorter
+    than `HashDigest<T>.Size` on .NET Standard 2.0. [[#1706], [#1815]]
+ -  Fixed `HashDigest<T>.DeriveFrom(ReadOnlySpan<byte>)` method's bug where
+    it had returned the wrong digest on .NET Standard 2.0. [[#1706], [#1815]]
```

Comment:
is it necessary?

Reply:
I thought so.  There are two patterns of situations, and these affect users in different ways.  One simply stops the subroutine (or the entire program), whereas other one can be a security flaw.